prime-router/src/main/kotlin/history/azure/DeliveryFacade.kt

```diff
@@ -104,6 +104,15 @@ class DeliveryFacade(
         )
     }
 
+    /**
+     * Get facilities for multiple deliveries in one call for a set of [reportIds]
```

Comment:
See https://kotlinlang.org/docs/kotlin-doc.html for kdoc.

Reply:
Updated!